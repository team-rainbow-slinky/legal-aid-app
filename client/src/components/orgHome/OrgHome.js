import React from 'react';
import Header from '../../containers/header/Header';
import Bookings from '../../containers/Bookings';

export default function OrgHome() {
  return (
    <>
      <Header/>
      <h1>HOME</h1>
      <Bookings />
    </>
  );
}
