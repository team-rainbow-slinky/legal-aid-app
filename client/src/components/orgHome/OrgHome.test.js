import React from 'react';
import OrgHome from './OrgHome';
import { shallow } from 'enzyme';

describe('OrgHome component', () => {
  it('renders the org home and displays bookings', () => {

    const wrapper = shallow(<OrgHome />);
    expect(wrapper).toMatchSnapshot();
  });
});
