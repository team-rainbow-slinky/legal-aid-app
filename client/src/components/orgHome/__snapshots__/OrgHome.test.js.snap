// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrgHome component renders the org home and displays bookings 1`] = `
<Fragment>
  <Connect(Header) />
  <h1>
    Organization Records
  </h1>
  <Connect(WithFetch) />
  <Footer />
</Fragment>
`;
